__tests__/handleErrors-tests.js

```diff
@@ -0,0 +1,149 @@
+// @flow
+import { fetchQuery } from '../__testUtils__/fetchQuery';
+import { queryMock } from '../__testUtils__/queryMock';
+
+
+describe('handleErrors', () => {
+  describe('printNoMockFoundError', () => {
+    it('should throw an error when there is no query name', async () => {
+      try {
```

Comment:
I'm not super happy with this pattern but I could not find a terser way to test exception messages that also caught when errors were not raised.  I tried to use a helper `expectErrorMessageToMatchSnapshot` function but errors would be swallowed as unhandled promises unless expectErrorMessageToMatchSnapshot was itself wrapped with a try / catch. Which defeated most of the purpose of the helper function.

Reply:
Yeah, I feel you... Async/await/promises is a mess when dealing with errors. I think it's just something we'll have to live with for now. I'm fine with it.